build.sbt

```diff
@@ -88,3 +89,180 @@ lazy val core = project
+        "repo1.maven.org",
+        "maven2",
+        "org",
+        "glassfish",
+        "jaxb",
+        "txw2",
+        "2.2.11",
+        "*"
+      ).mkString(java.io.File.separator),
+      Seq(
+        s"${csrCacheDirectory.value}",
+        "https",
+        "repo1.maven.org",
+        "maven2",
+        "relaxngDatatype",
+        "relaxngDatatype",
+        "20020414",
+        "*"
+      ).mkString(java.io.File.separator),
+      Seq(
+        s"${csrCacheDirectory.value}",
+        "https",
+        "repo1.maven.org",
+        "maven2",
+        "com",
+        "sun",
+        "xsom",
+        "xsom",
+        "20140925",
+        "*"
+      ).mkString(java.io.File.separator),
+      Seq(
+        s"${csrCacheDirectory.value}",
+        "https",
+        "repo1.maven.org",
+        "maven2",
+        "com",
+        "sun",
+        "xml",
+        "bind",
+        "external",
+        "rngom",
+        "2.2.11",
+        "*"
+      ).mkString(java.io.File.separator),
+      Seq(
+        s"${csrCacheDirectory.value}",
+        "https",
+        "repo1.maven.org",
+        "maven2",
+        "com",
+        "sun",
+        "istack",
+        "istack-commons-runtime",
+        "2.21",
+        "*"
+      ).mkString(java.io.File.separator),
+      Seq(
+        s"${csrCacheDirectory.value}",
+        "https",
+        "repo1.maven.org",
+        "maven2",
+        "org",
+        "glassfish",
+        "jaxb",
+        "codemodel",
+        "2.2.11",
+        "*"
+      ).mkString(java.io.File.separator),
+      Seq(
+        s"${csrCacheDirectory.value}",
+        "https",
+        "repo1.maven.org",
+        "maven2",
+        "org",
+        "glassfish",
+        "jaxb",
+        "jaxb-core",
+        "2.2.11",
+        "*"
+      ).mkString(java.io.File.separator),
+      Seq(
+        s"${csrCacheDirectory.value}",
+        "https",
+        "repo1.maven.org",
+        "maven2",
+        "javax",
+        "xml",
+        "bind",
+        "jaxb-api",
+        "2.1",
+        "*"
+      ).mkString(java.io.File.separator),
+      Seq(
+        s"${csrCacheDirectory.value}",
+        "https",
+        "repo1.maven.org",
+        "maven2",
+        "com",
+        "sun",
+        "istack",
+        "istack-commons-tools",
+        "2.21",
+        "*"
+      ).mkString(java.io.File.separator),
+      Seq(
+        s"${csrCacheDirectory.value}",
+        "https",
+        "repo1.maven.org",
+        "maven2",
+        "org",
+        "glassfish",
+        "jaxb",
+        "jaxb-xjc",
+        "2.2.11",
+        "*"
+      ).mkString(java.io.File.separator),
+      Seq(
+        s"${csrCacheDirectory.value}",
+        "https",
+        "repo1.maven.org",
+        "maven2",
+        "com",
+        "sun",
+        "xml",
+        "bind",
+        "jaxb-impl",
+        "2.2.11",
+        "*"
+      ).mkString(java.io.File.separator)
+    ).mkString(java.io.File.pathSeparator),
+    Compile / xjc / sources := Seq(file("resources/xsd")),
```

Comment:
Should these files be in the `server/sbtXjc` subproject? Though, maybe I'm not sure what the sbtXjc subproject is for? That directory doesn't actually exist?